katello-installer/katello-installer-base.spec

```diff
@@ -99,6 +105,9 @@ sed -ri 'sX\:installer_dir.*$X:installer_dir: %{_datadir}/katello-installer-base
 sed -ri 'sX\:module_dirs.*$X:module_dirs: \["%{_datadir}/katello-installer-base/modules", "%{_datadir}/foreman-installer/modules"\]Xg' config/*.yaml
 sed -ri 'sX\:hook_dirs.*$X:hook_dirs: \["%{_datadir}/katello-installer-base/hooks"\]Xg' config/*.yaml
 
+# generate parser-cache
+kafo-export-params -c config/katello.yaml -f parsercache --no-parser-cache -o %{_datadir}/foreman-installer/katello.yaml
```

Comment:
By rake build do you mean rake pkg:generate_source ? They run that to
generate source in Jenkins the same way for nighties afaik.

On Jul 29, 2016 12:33 PM, "Stephen Benjamin" <notifications@github.com>
wrote:

> In katello-installer/katello-installer-base.spec
> <https://github.com/Katello/katello-packaging/pull/260#discussion_r72821687>
> :
>
> > @@ -99,6 +105,9 @@ sed -ri 'sX\:installer_dir.*$X:installer_dir: %{_datadir}/katello-installer-base
> >  sed -ri 'sX\:module_dirs.*$X:module_dirs: \["%{_datadir}/katello-installer-base/modules", "%{_datadir}/foreman-installer/modules"\]Xg' config/*.yaml
> >  sed -ri 'sX\:hook_dirs.*$X:hook_dirs: \["%{_datadir}/katello-installer-base/hooks"\]Xg' config/*.yaml
> >
> > +# generate parser-cache
> > +kafo-export-params -c config/katello.yaml -f parsercache --no-parser-cache -o %{_datadir}/foreman-installer/katello.yaml
>
> We have a similar rake task in our katello-installer so I would think it
> should be added there, the only difference is they run the rake build in
> the RPM spec, and we do it out of band somewhere else (jenkins?)
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/Katello/katello-packaging/pull/260/files/3bc806c3b8155da00843de101b96ea3356c5f4dd#r72821687>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAR585xzJ9pM5ekb402_03OgsuUsBbxOks5qaithgaJpZM4JXbUe>
> .
>